data/json/vehicles/vans_busses.json

```diff
@@ -1791,21 +1789,8 @@
       { "x": -8, "y": 3, "part": "plating_steel" }
     ],
     "items": [
-      { "x": -1, "y": 0, "chance": 7, "item_groups": [ "contraband" ] },
-      { "x": 0, "y": 0, "chance": 20, "items": [ "id_science_transport_1" ] },
```

Comment:
Why delete the science ID chance? This was added just a month ago to give some ways to legally enter the new labs.

Reply:
Specifically this one would be considered to be on one of the guards as these prisoners were being rerouted for experimentation.
`{ "x": 0, "y": 0, "chance": 20, "items": [ "id_science_transport_1" ] },`

Reply:
I'll give it the same chance of spawning except in the front compartment somewhere if that's okay with you guys. :)

Reply:
It sounds fine to me!

Reply:
Sounds good. :)

Reply:
Added it to the box in the front cabin, @Termineitor244, as well as in the same spot on the prison van but with a smaller chance of spawning. I feel those boxes should both be lock boxes really, (so behind a lockpicking skill-check, i guess) but would this make the card not accessible enough?

Reply:
It would be even better if its behind a lock! It would give even more of a sense of "This is something important, I should keep it with me". And a few locks are never enough deterrent for an experienced player.